[2024-05-08T04:50:21.266+0000] {processor.py:161} INFO - Started process (PID=165) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:21.270+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:50:21.275+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:21.275+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:21.577+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:21.959+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:21.959+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:50:22.001+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:22.000+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:50:22.033+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.802 seconds
[2024-05-08T04:50:37.151+0000] {processor.py:161} INFO - Started process (PID=191) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:37.158+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:50:37.160+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:37.159+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:37.417+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:50:37.619+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:37.618+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:50:37.767+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:50:37.766+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:50:37.837+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.759 seconds
[2024-05-08T04:51:08.138+0000] {processor.py:161} INFO - Started process (PID=216) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:08.145+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:51:08.150+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:08.150+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:08.355+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:08.547+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:08.546+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:51:08.771+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:08.770+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:51:08.811+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.719 seconds
[2024-05-08T04:51:32.487+0000] {processor.py:161} INFO - Started process (PID=242) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:32.490+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:51:32.493+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:32.492+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:32.665+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:51:33.105+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:33.105+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:51:33.188+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:51:33.188+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:51:33.280+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.814 seconds
[2024-05-08T04:52:04.129+0000] {processor.py:161} INFO - Started process (PID=267) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:04.131+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:52:04.134+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:04.133+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:04.295+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:04.356+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:04.356+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:52:04.411+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:04.410+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:52:04.455+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.338 seconds
[2024-05-08T04:52:08.499+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:08.506+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:52:08.509+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:08.508+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:08.630+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:08.878+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:08.877+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:52:08.931+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:08.931+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:52:08.988+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.514 seconds
[2024-05-08T04:52:39.891+0000] {processor.py:161} INFO - Started process (PID=298) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:39.894+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:52:39.899+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:39.898+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:40.119+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:52:40.180+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:40.179+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:52:40.224+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:52:40.223+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:52:40.280+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.419 seconds
[2024-05-08T04:53:10.700+0000] {processor.py:161} INFO - Started process (PID=324) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:10.702+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:53:10.704+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:10.703+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:10.775+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:10.828+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:10.827+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:53:10.873+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:10.873+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:53:10.904+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.214 seconds
[2024-05-08T04:53:12.905+0000] {processor.py:161} INFO - Started process (PID=329) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:12.906+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:53:12.908+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:12.907+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:13.023+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:13.252+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:13.251+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:53:13.281+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:13.281+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:53:13.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.429 seconds
[2024-05-08T04:53:21.147+0000] {processor.py:161} INFO - Started process (PID=334) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:21.149+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:53:21.152+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:21.151+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:21.322+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:21.363+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:21.362+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:53:21.449+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:21.448+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:53:21.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.392 seconds
[2024-05-08T04:53:52.461+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:52.463+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:53:52.465+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:52.464+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:52.545+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:53:52.770+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:52.770+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:53:52.809+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:53:52.808+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:53:52.850+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.400 seconds
[2024-05-08T04:54:23.028+0000] {processor.py:161} INFO - Started process (PID=387) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:23.029+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:54:23.031+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:54:23.031+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:23.110+0000] {processor.py:840} INFO - DAG(s) 'mysql_table_dump' retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:23.168+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:54:23.167+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-08T04:54:23.231+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:54:23.230+0000] {dag.py:3954} INFO - Setting next_dagrun for mysql_table_dump to None, run_after=None
[2024-05-08T04:54:23.282+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.268 seconds
[2024-05-08T04:54:47.054+0000] {processor.py:161} INFO - Started process (PID=413) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:47.056+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:54:47.062+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:54:47.060+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:47.136+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:54:47.116+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:54:47.139+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:54:47.222+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.189 seconds
[2024-05-08T04:55:17.913+0000] {processor.py:161} INFO - Started process (PID=440) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:17.915+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:55:17.916+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:55:17.916+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:17.945+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:55:17.943+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:55:17.946+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:17.974+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.072 seconds
[2024-05-08T04:55:48.852+0000] {processor.py:161} INFO - Started process (PID=466) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:48.855+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:55:48.859+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:55:48.858+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:48.965+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:55:48.962+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:55:48.967+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:55:49.004+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.189 seconds
[2024-05-08T04:56:19.164+0000] {processor.py:161} INFO - Started process (PID=491) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:19.165+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:56:19.167+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:56:19.167+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:19.194+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:56:19.192+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:56:19.201+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:19.239+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.086 seconds
[2024-05-08T04:56:50.154+0000] {processor.py:161} INFO - Started process (PID=517) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:50.157+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:56:50.159+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:56:50.158+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:50.184+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:56:50.182+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:56:50.185+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:56:50.252+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.108 seconds
[2024-05-08T04:57:20.765+0000] {processor.py:161} INFO - Started process (PID=543) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:20.766+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:57:20.767+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:20.767+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:20.787+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:20.785+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:57:20.789+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:20.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.076 seconds
[2024-05-08T04:57:22.830+0000] {processor.py:161} INFO - Started process (PID=548) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:22.831+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:57:22.833+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:22.832+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:22.858+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:22.856+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:57:22.859+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:22.888+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.071 seconds
[2024-05-08T04:57:53.263+0000] {processor.py:161} INFO - Started process (PID=574) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:53.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:57:53.269+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:53.268+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:53.373+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:57:53.371+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:57:53.374+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:57:53.410+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.180 seconds
[2024-05-08T04:58:23.938+0000] {processor.py:161} INFO - Started process (PID=600) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:23.939+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:58:23.941+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:58:23.940+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:23.963+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:58:23.962+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:58:23.964+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:23.991+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.063 seconds
[2024-05-08T04:58:54.705+0000] {processor.py:161} INFO - Started process (PID=626) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:54.706+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:58:54.708+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:58:54.707+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:54.736+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:58:54.734+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:58:54.737+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:58:54.761+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.065 seconds
[2024-05-08T04:59:25.340+0000] {processor.py:161} INFO - Started process (PID=652) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:25.342+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:59:25.344+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:59:25.344+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:25.373+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:59:25.371+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:59:25.374+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:25.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.070 seconds
[2024-05-08T04:59:56.283+0000] {processor.py:161} INFO - Started process (PID=677) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:56.286+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T04:59:56.289+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:59:56.288+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:56.328+0000] {logging_mixin.py:188} INFO - [2024-05-08T04:59:56.325+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T04:59:56.330+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T04:59:56.376+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.118 seconds
[2024-05-08T05:00:26.490+0000] {processor.py:161} INFO - Started process (PID=703) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:26.491+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:00:26.493+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:00:26.493+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:26.525+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:00:26.523+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:00:26.526+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:26.559+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.081 seconds
[2024-05-08T05:00:57.472+0000] {processor.py:161} INFO - Started process (PID=729) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:57.474+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:00:57.475+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:00:57.475+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:57.504+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:00:57.503+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:00:57.505+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:00:57.534+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.071 seconds
[2024-05-08T05:01:28.272+0000] {processor.py:161} INFO - Started process (PID=761) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:28.276+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:01:28.279+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:01:28.278+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:28.392+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:01:28.389+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:01:28.405+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:28.538+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.292 seconds
[2024-05-08T05:01:58.905+0000] {processor.py:161} INFO - Started process (PID=787) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:58.906+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:01:58.909+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:01:58.908+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:58.945+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:01:58.943+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:01:58.947+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:01:58.983+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.094 seconds
[2024-05-08T05:02:29.684+0000] {processor.py:161} INFO - Started process (PID=813) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:29.686+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:02:29.691+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:02:29.690+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:29.755+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:02:29.752+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:02:29.758+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:29.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.170 seconds
[2024-05-08T05:02:48.676+0000] {processor.py:161} INFO - Started process (PID=833) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:48.685+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:02:48.688+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:02:48.687+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:48.752+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:02:48.749+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:02:48.754+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:02:48.813+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.177 seconds
[2024-05-08T05:03:19.059+0000] {processor.py:161} INFO - Started process (PID=859) to work on /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:03:19.061+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dump_load_dag.py for tasks to queue
[2024-05-08T05:03:19.063+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:03:19.062+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:03:19.098+0000] {logging_mixin.py:188} INFO - [2024-05-08T05:03:19.093+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/dump_load_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dump_load_dag.py", line 34
    bash_command=,
    ^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-05-08T05:03:19.099+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dump_load_dag.py
[2024-05-08T05:03:19.246+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dump_load_dag.py took 0.207 seconds
